src/wax_contract.cpp

```diff
@@ -232,6 +234,13 @@ CONTRACT WAX_CONTRACT_NAME: public contract {
             return default_value;
         return it->value;
     }
+
+    uint64_t generate_next_index() {
+        const uint64_t entry_name{"jobid.index"_n.value};
+        int64_t index_val = get_config(entry_name, 0);
```

Comment:
Maybe index_val should be uint64_t with a static cast here?

Reply:
Since values in config table are stored as `int64_t`, it makes sense that `get_config()` returns a `int64_t`, then it's fine to _receive_ this value in a `int64_t` variable (`index_val`). Actually I think we don't need the cast on the line below, since index_vale is already a `int64_t` type.

Reply:
indeed. That's why I thought you meant uint64_t in the previous line - due to the cast in the following one